data_steward/cdr_cleaner/cleaning_rules/deid/fitbit_pid_rid_map.py

```diff
@@ -0,0 +1,85 @@
+"""
+DEID rule to change PIDs to RIDs for Fitbit tables
+"""
+# Python Imports
+import logging
+
+# Project imports
+from cdr_cleaner.cleaning_rules.deid.pid_rid_map import PIDtoRID
+from common import FITBIT_TABLES
+
+LOGGER = logging.getLogger(__name__)
+
+ISSUE_NUMBERS = ['DC-1000']
+
+
+class FitbitPIDtoRID(PIDtoRID):
```

Comment:
I'm a little confused at what this module is doing?

Reply:
Sure, this module doesn't do anything other than supply the `affected_tables` and `issue_numbers`. The rest is handled by the parent `PIDtoRID` class. The empty `setup_validation` is because it is not implemented, but I can remove that to make it simple.

Reply:
Ah, okay. That makes sense now!